CHANGELOG.md

```diff
@@ -5,6 +5,9 @@ however since API is already quite stable we don't expect too much breaking chan
 If we missed a note on some change or you have a questions on migrating from old version, 
 feel free to ask us and community.
 
+## 0.1.8 (future)
```

Comment:
Don't you think we shall provide more details on site as well?

Reply:
Might be a good idea, by where? FAQ?

Reply:
not sure, shouldn't it be in http://typeorm.io/#/supported-platforms ?

Reply:
btw this will land in 0.1.7 which Im planning to release later today